test/src/commands/test/test_test.dart

```diff
@@ -413,6 +413,44 @@ void main() {
       ).called(1);
     });
 
+    test('displays required precision see why coverage was not met', () async {
+      when<dynamic>(() => argResults['coverage']).thenReturn(true);
+      when<dynamic>(() => argResults['min-coverage']).thenReturn('100');
+      const exception = MinCoverageNotMet(99.999995);
+      when(
+        () => flutterTest(
+          cwd: any(named: 'cwd'),
+          recursive: any(named: 'recursive'),
+          collectCoverage: any(named: 'collectCoverage'),
+          optimizePerformance: any(named: 'optimizePerformance'),
+          minCoverage: any(named: 'minCoverage'),
+          excludeFromCoverage: any(named: 'excludeFromCoverage'),
+          arguments: any(named: 'arguments'),
+          logger: any(named: 'logger'),
+          stdout: any(named: 'stdout'),
+          stderr: any(named: 'stderr'),
+        ),
+      ).thenThrow(exception);
+      final result = await testCommand.run();
+      expect(result, equals(ExitCode.unavailable.code));
+      verify(
+        () => flutterTest(
+          optimizePerformance: true,
+          collectCoverage: true,
+          arguments: defaultArguments,
+          minCoverage: 100,
+          logger: logger,
+          stdout: logger.write,
+          stderr: logger.err,
+        ),
+      ).called(1);
+      verify(
+        () => logger.err(
+          'Expected coverage >= 100.000000% but actual is 99.999995%.',
```

Comment:
I would still prefer the 100 to stay the same (`100.00%`), the actual is where the decimal point matters.

Reply:
I did this because there could be a case where someone has min coverage going to more than two decimal places, like `99.004`. In that case, if the coverage was 99.003, then the message would be `Expected coverage >= 99.00% but actual is 99.003%.`

Obviously it's an edge case, but I don't think we should round either value if it could make the error messaging more confusing. Plus, in most cases the coverage won't be that close and it will display 100.00% anyway.